metadata_tools/common.py

```
@@ -0,0 +1,224 @@
+                    default=False,
+                    help='''If given, labels are generated for existing files.''')
+    gr.add_argument('--pattern', '-p', type=str, metavar='pattern',
+                    help='''Glob pattern to select files.''')
+
+    # Return parser
+    return parser
+
+##########################################################################################
+# Table class
+##########################################################################################
+class Table(object):
```

Ruff (0.14.10)

Class `Table` inherits from `object`

Remove `object` inheritance

(UP004)